PluginExamples/Sources/Import/Foo/ImportFoo.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto3";
+
+import "Bar/ImportBar.proto";
+
+message ImportFoo {
```

Comment:
nit: remove `Import` it makes it sound like the type is importing `Foo` when it's actually importing a Bar